cassandra/tablets.py

```diff
@@ -51,7 +58,42 @@ def get_tablet_for_key(self, keyspace, table, t):
             return tablet[id]
         return None
 
+    def drop_tablet(self, keyspace: str, table: str = None):
+        print("drop tablets for ", keyspace, table)
+        with self._lock:
+            if table is not None:
+                self._tablets.pop((keyspace, table), None)
+                return
+
+            to_be_deleted = []
+            for key in self._tablets.keys():
+                if key[0] == keyspace:
+                    to_be_deleted.append(key)
+
+            for key in to_be_deleted:
+                del self._tablets[key]
```

Comment:
🔧 `drop_tablets` is a better name - we are dropping all tablets associated with particular ks / table, not just a single tablet.

Reply:
renames